src/components/ResultDialog.vue

```diff
@@ -10,7 +10,7 @@
       <div v-if="message" class="transaction-info">{{ message }}</div>
       <!-- This link was hidden due to PSS-205 task. We'll return it back later.  -->
       <!-- <s-icon name="external-link" @click="handleTransactionInfo" /> -->
-      <!-- TODO: Add correct icons and add functionality -->
+      <!-- Add correct icons and add functionality -->
```

Comment:
Why did you remove TODO's? It's easy to lose appropriate places without them(((

Reply:
Cuz we didn't use this component. I've added another todo like 'we don't use it'

Reply:
@alexnatalia I don't want to remove this component. Probably, we'll use it later... So, I've updated todo :)

Reply:
Could we remove it if so (it seems that we decided to skip this part of notification)?

Reply:
We will but I want to be sure, let's do it later, ok?

Reply:
ok, sure